docs/excel/custom-functions-json.md

```diff
@@ -150,12 +151,12 @@ The `functions` property is an array of custom function objects. The following t
 
 The `options` object enables you to customize some aspects of how and when Excel executes the function. The following table lists the properties of the `options` object.
 
-| Property          | Data type | Required                               | Description                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                |
+| Property          | Data type | Required                               | Description |
 | :---------------- | :-------- | :------------------------------------- | :--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
```

Comment:
At least 3 dashes are needed but you don't need the number of dashes to match the maximum number of characters in a subsequent row for that column.
```suggestion
| :---------------- | :-------- | :------------------------------------- | :---------- |
```

Reply:
Sweet! Thanks, I wondered why there were so many dashes here.